handsontable/test/e2e/core/selectColumns.spec.js

```diff
@@ -50,16 +50,19 @@ describe('Core.selectColumns', () => {
       `).toBeMatchToSelectionPattern();
   });
 
-  it('should mark non-contiquous selection when CTRL key is pressed', () => {
+  // TODO: After changes introduced in Handsontable 12.0.0 we handle shortcuts only by listening Handsontable.
+  // Please keep in mind not passing test is consequence of bug #7290.
+  xit('should mark non-contiquous selection when CTRL key is pressed', () => {
```

Comment:
```suggestion
  it('should mark non-contiquous selection when CTRL key is pressed', () => {
```
The test works after adding the `listen()` call. IMO the `selectColumns` does not change the table listening state by design.

Reply:
Snippet that fixes the test
```js
    listen();
    selectColumns(2);
    keyDown('control');
    selectColumns(0);
    keyUp('control');
```

Reply:
As in https://github.com/handsontable/handsontable/pull/8955#discussion_r811899513.